Update Regulation A6e to WRC understanding

mirandapedro opened the pull request:
Update Regulation A6e to WRC understanding by specifying the requirements for lifting the time penalty for touching the puzzle after releasing it.

Fixes #200, #534, [this](https://forum.worldcubeassociation.org/t/regulation-a6e-revisement/3225/3) Forum post and makes the adjustment mentioned on Incident Log [#23](https://www.worldcubeassociation.org/incidents/23).

There are three main issues that should be addressed on this update:
1. When can the competitor touch the puzzle after stopping the timer - the current wording "until the judge has inspected the puzzle" is not very clear;
2. On what circumstances can the time penalty for touching the puzzle after stopping the timer be lifted; and
3. Whether a puzzle falling on the competitor's lap should still be penalized - currently made clear by Guideline A6e+.

The solution for each one of these issues is:

1. Making it so the competitor can only touch the puzzle after the attempt ends (after the competitor has signed the scoresheet, as per Regulation A7c) or after being explicitly allowed to do so by the judge.

2. Instituting the following three requirements for lifting the time penalty for touching the puzzle after releasing it (also present in the above mentioned Incident Log):
- The touch must happen immediately after stopping the timer - this Regulation aims at being lenient toward instinctual or automatic reactions (like the one made on [this solve](https://www.youtube.com/watch?v=YSo8w1yPetE&feature=youtu.be&t=1h22m12s) by Lucas Etter), and not touch to the puzzle that happens when the competitor is calm or can think things through;
- The touch must be brief - for the same reason as above, a prolonged touch to the puzzle constitutes a clear violation of Regulation A6e; and
- The touch must not affect the state of the puzzle - if it does, the attempt should be disqualified, as per the proposed Regulation A6e1. To be able to determine whether this happened or not, the WCA Delegate may use video evidence, as per Regulation 11f.

This commit also adds the explicit need for WCA Delegate discretion for applying the exception, as judges shouldn't be the ones making these decisions.

I believe an update do A6e shouldn't get into the merit of the touch being intentional or not. Besides it being subjective to determine intention (you could argue, for example, that Lucas' touch in the above-linked video was intentional), it isn't the best criteria - sometimes an unintentional touch should be penalized (e.g. a puzzle falling on a competitor's lap) and an "intentional" (or instinctual) touch shouldn't.

3. As a puzzle falling on a competitor's lap does not configure a brief touch - thus not meeting the requirements defined in Regulation A6e2 - it should still be penalized. That's why Guideline A6e+ should still stand as it is.

Finally, there shouldn't be any discretion in determining whether a violation of Regulation A6e results in a DNF or a +2. That's why these two situations were treated separately and made different.

Commit:
Update Regulation A6e to WRC understanding

Update Regulation A6e to WRC understanding by specifying the requirements for lifting the time penalty for touching the puzzle after releasing it.

Fixes #200, #534, [this](https://forum.worldcubeassociation.org/t/regulation-a6e-revisement/3225/3) Forum post and makes the adjustment mentioned on Incident Log [#23](https://www.worldcubeassociation.org/incidents/23).

There are three main issues that should be addressed on this update:
1. When can the competitor touch the puzzle after stopping the timer - the current wording "until the judge has inspected the puzzle" is not very clear;
2. On what circumstances can the time penalty for touching the puzzle after stopping the timer be lifted; and
3. Whether a puzzle falling on the competitor's lap should still be penalized - currently made clear by Guideline A6e+.

The solution for each one of these issues is:

1. Making it so the competitor can only touch the puzzle after the attempt ends (after the competitor has signed the scoresheet, as per Regulation A7c) or after being explicitly allowed to do so by the judge.

2. Instituting the following three requirements for lifting the time penalty for touching the puzzle after releasing it (also present in the above mentioned Incident Log):
- The touch must happen immediately after stopping the timer - this Regulation aims at being lenient toward instinctual or automatic reactions (like the one made on [this solve](https://www.youtube.com/watch?v=YSo8w1yPetE&feature=youtu.be&t=1h22m12s) by Lucas Etter), and not touch to the puzzle that happens when the competitor is calm or can think things through;
- The touch must be brief - for the same reason as above, a prolonged touch to the puzzle constitutes a clear violation of Regulation A6e; and
- The touch must not affect the state of the puzzle - if it does, the attempt should be disqualified, as per the proposed Regulation A6e1. To be able to determine whether this happened or not, the WCA Delegate may use video evidence, as per Regulation 11f.

This commit also adds the explicit need for WCA Delegate discretion for applying the exception, as judges shouldn't be the ones making these decisions.

I believe an update do A6e shouldn't get into the merit of the touch being intentional or not. Besides it being subjective to determine intention (you could argue, for example, that Lucas' touch in the above-linked video was intentional), it isn't the best criteria - sometimes an unintentional touch should be penalized (e.g. a puzzle falling on a competitor's lap) and an "intentional" (or instinctual) touch shouldn't.

3. As a puzzle falling on a competitor's lap does not configure a brief touch - thus not meeting the requirements defined in Regulation A6e2 - it should still be penalized. That's why Guideline A6e+ should still stand as it is.

Finally, there shouldn't be any discretion in determining whether a violation of Regulation A6e results in a DNF or a +2. That's why these two situations were treated separately and made different.

Commit:
Fix mistakes in initial PR

Fixes trailling whitespace and missing link in a reference to Regulation 11f.

xsrvmy commented:
Wording problem: the time penalty *may not* be applied - "may not" is not in RFC2119. Probably should be "should not" or "must not".

---

Rest is comment on the proposal itself:

Is there a reason why the WRC has decided to make A6e stricter when almost everyone thinks its unnecessarily strict as it is? Wasn't there a proposal to loosen A6e so that the +2 is not given as long as there is no potential misalignment penalty that need to be resolved? Why isn't allowing the competitor to touch the puzzle after the judge signs enough?

I think I need to voice a pretty strong objection to the first criteria:
There are at least three common actions that can lead to the puzzle being accidentally touched: reactions, reaching for the camera, signing (this one is new). The first criteria does not cover the third case at all, and only partially covers the second case. The competitor might also accidentally make contact with the puzzle while the judge is holding it.

Here is an example where the cube is touched when picking up the camera (former 6x6 WR): https://www.youtube.com/watch?v=RrSGmPKRdr4

Also two questions arise about signing:
- What if the competitor signs, and while signing, touches the puzzle? The regs do not allow adding a +2 to the scoresheet at this point.
- Does this void A7c4+?

KitClement commented:
Is there any reason the new A6e2 is at the discretion of the delegate? The delegate is typically not the one that will be viewing the solve in the first place, it seems more reasonable to have this at the discretion of the judge.

SAuroux commented on wca-regulations.md:
```diff
@@ -405,7 +405,9 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - A6d) The competitor must stop the timer using both hands, placed flat on the sensors with palms down. Penalty: time penalty (+2 seconds).
-    - A6e) After releasing the puzzle, the competitor must not touch or move the puzzle until the judge has inspected the puzzle. Penalty: disqualification of the attempt (DNF). Exception: If no moves have been applied, a time penalty (+2 seconds) may be assigned instead, at the discretion of the judge.
+    - A6e) After stopping the timer, the competitor must not touch or move the puzzle until the attempt has ended (see [Regulation A7c](regulations:regulation:A7c)) or after the judge has explicitly allowed them to do so.
+            - A6e1) Penalty for moving the puzzle after stopping the timer: disqualification of the attempt (DNF).
```
I am certain that _moving the puzzle_ is not a valid synonym for _applying moves to the puzzle_.

SAuroux commented:
I see a lot of issues with this proposal and I really do not think that the current wording is practically applicable at all. Most of all, changing the scope of this regulation from _until the puzzle has been inspected_ to _until the end of the attempt_ is a significant scope extension that introduces a lot of additional strictness which I believe to be unnecessary. 

Especially, I do not think that all implications have been thought through well enough. For instance:
* Pushing aside the puzzle to create space on the mat for signing as a competitor is now a DNF according to the current wording of A6e1 as the puzzle has been _moved_ out of place. But also after fixing the wording error in A6e1 to _applying moves to the puzzle_, the same action is still a clear +2 according to A6e2. Surely, this cannot be intended.
* Situation being absolutely common these days would now be clear DNFs, e.g.:
  * A competitor aligning their puzzle from +2 state to solved state before signing the result (with +2) already correctly recorded and signed by the judge
  * A competitor swiftly practicing a previously messed up PLL before signing a result already correctly recorded and signed by the judge

For those reasons and also to prevent adding unnecessary strictness, I think that the regulation's scope should not be extended. If anything, it should rather be narrowed. Two weeks ago I shared my opinion in the corresponding [WCA form thread](https://forum.worldcubeassociation.org/t/regulation-a6e-revisement/3225/7?u=sebastien_auroux) and questioned where is the sense in applying a +2 penalty if it is already clear that the final state of the puzzle has not been affected? 

I have always been convinced that the ultimate intention behind regulation A6e is to ensure that if a competitor prevents the judge from inspecting the final state of the puzzle, they get DNF'd. Isn't this what we actually want? If yes, I really do not think that so much has to be changed starting from the existing regulation. Here is my attempt:

> A6e) After releasing the puzzle and stopping the timer, the competitor should not interact with the puzzle until the judge has inspected the puzzle and determined its final state. Penalty for any interaction of the competitor with the puzzle due to which the judge is unable to capture the final state of the puzzle when the timer was stopped: disqualification of the attempt (DNF), at the discretion of the judge.

EdHollingdale commented:
I completely agree with @SAuroux here and I like the wording he has proposed. Extending the regulation in this way adds more unnecessary strictness, and the +2 rule is confusing.

I also don't get the +2 rule and exception - it feels like it has been worded to not change too much while fitting with prior WRC rulings. In what instances would the +2 rule be applied? If the judge thinks there was a move applied then it is DNF, if there was no move applied and the competitor touched after stopping the solve then it is most likely no penalty. Or at least I would say that if not for the "immediately" which is why it like it is just there to fix a noteworthy WRC decision. 

Applying a few real world tests should show the weakness of the proposed change: what if I brush the cube while reaching to sign the scorecard - is that at +2 now? What if I forget to sign the scorecard and the score-taker finds me later, is that an automatic +2? Or did I touch the cube "immediately" after enough for it to be covered by the exception? In which case, why even have the +2 as a rule?

Commit:
Improve the wording of Regulation A6e after feedback

This commit improves the proposed wording of Regulations A6e, A6e1, and A6e2 after WCA Staff and Community Feedback.

Summary of changes:
1. Change back the moment until the competitor is forbidden from touching the puzzle to when the "judge has inspected the puzzle", as it is currently
2. Improve the wording of "moving" to "applying a move"
3. Change WCA Delegate discretion to judge discretion in regards to who can lift the penalty of touching the puzzle
4. Change the wording of "may not" to "may" in Regulation A6e2

On a general note, the WRC appreciated the proposal that consists of applying no penalty if the judge was able to determine the state of the puzzle before the competitor touched it and applying a DNF if the judge wasn't able to do it.

However, we decided against this proposal as it lessens the punishment/discouragement for competitors that touch the puzzle after finishing their attempt. In our view, this would lead to more and more competitors "greying" the line of when they can and cannot touch the puzzle, which is something we definitely want to steer away from.

Therefore, we agreed on maintaining a +2 in these Regulations, encouraging competitors to not touch the puzzle until the judge has been able to clearly inspect the puzzle.

With that being said, here are the details on the specific changes of this commit:

1. After WCA Staff feedback, it was clear that requiring competitors not to touch the puzzle until they had signed the scoresheet was unnecessarily strict and had no good reason to be this way. I had attempted to solve two problems at once but ended up going in an undesired direction.

2. This improvement comes from @SAuroux very accurately pointing out that "moving" a puzzle is not the same as "applying a move" to it. The wording change, therefore, is meant to make the proposed wording clearer.

3. This change is a result of @KitClement correctly highlighting that, in most cases, it is not the WCA Delegate that will be deciding to lift the +2 penalty, but the judge.

4. This modification was made after @xsrvmy indicated that "may not" is not up to RFC 2119 standard.

Thank you to everyone who contributed to this update!

mirandapedro commented:
Thank you to everyone who contributed to this PR!

Ivan-Brigidano committed:
Merge branch 'draft' into mirandapedro-A6e-update
